src/relay/op/nn/convolution.cc

```diff
@@ -74,11 +75,22 @@ bool Conv2DRel(const Array<Type>& types,
   if (param->kernel_size.defined() && param->channels.defined()) {
     CHECK_EQ(param->kernel_size.size(), 2);
     CHECK_EQ(param->dilation.size(), 2);
-    Array<IndexExpr> wshape(
-       {param->channels,
-         dshape_nchw[1] / param->groups,
-         param->kernel_size[0],
-         param->kernel_size[1]});
+    Array<IndexExpr> wshape;
+
+    if (tvm::ir::Equal(param->channels, param->groups)) {
```

Comment:
How about adding a comment to note this is the case for depthwise convolution?

Reply:
updated